src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -1683,7 +1702,7 @@ public enum LegendMaker implements LogicCardInfo {
           energyCost C, C
           onAttack {
             damage 20
-            if (self.cards.findAll { it.name.contains("React Energy") }) {
+            if (self.cards.any{ it.name == "React Energy" }) {
```

Comment:
I think this is the proper way to check, and may need applied to some of the other React Energy checks.

Reply:
Thanks for the reminder, just corrected all missing instances of it 👍